CommandLine/libEGM/egm-events.h

```diff
@@ -0,0 +1,27 @@
+/* Copyright (C) 2020 Greg Williamson, Josh Ventura
+***
+*** This file is a part of the ENIGMA Development Environment.
+***
+*** ENIGMA is free software: you can redistribute it and/or modify it under the
+*** terms of the GNU General Public License as published by the Free Software
+*** Foundation, version 3 of the license or any later version.
+***
+*** This application and its source code is distributed AS-IS, WITHOUT ANY
+*** WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+*** FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+*** details.
+***
+*** You should have received a copy of the GNU General Public License along
+*** with this code. If not, see <http://www.gnu.org/licenses/>
+*/
+
```

Comment:
I'm not sure this really needs its own header, as opposed to just going in the egm-read/write headers. But I won't quibble. The name's just a little jarring because we had an entire branch called EgmEvents that was about the event descriptors themselves, rather than the event code protos.